AUTHORS

```diff
@@ -233,7 +233,6 @@ Prasoon Shukla <prasoon92.iitr@gmail.com>
 Stefen Yin <zqyin@ucdavis.edu>
 Thomas Hisch <t.hisch@gmail.com>
 Madeleine Ball <mpball@gmail.com>
-Case Van Horsen <casevh@gmail.com>
```

Comment:
For Line 236, I found duplicate at Line 42 : [here](https://github.com/sympy/sympy/blob/master/AUTHORS#L42)
For Line 477, I found duplicate at Line 393 : [here](https://github.com/sympy/sympy/blob/master/AUTHORS#L393)
For Line 525, I found duplicate at Line 486 : [here](https://github.com/sympy/sympy/blob/master/AUTHORS#L486)
For Line 527., I found duplicate at Line 528 : [here](https://github.com/sympy/sympy/blob/master/AUTHORS#L528) (See email)
I though these duplicates went unnoticed previously. Please let me know if any(or all) of these should be reverted back?